scripts/check-dialyzer.escript

```diff
@@ -12,9 +12,12 @@
 
 main(Args) ->
     case [Arg || Arg <- Args,
-                 is_ebin_dir(Arg)
-                     orelse is_beam(Arg)
-                     orelse is_erl(Arg)
+                 (
+                   is_ebin_dir(Arg)
+                   orelse is_beam(Arg)
+                   orelse is_erl(Arg)
+                 ) andalso
+                     not is_test(Arg)
```

Comment:
Can this go first so we short circuit all the other conditions if it is a test beam?